client/ayon_deadline/plugins/publish/global/submit_publish_job.py

```diff
@@ -432,6 +432,7 @@ def process(self, instance):
             publish_job.update({"audio": audio_file})
 
         with open(metadata_path, "w") as f:
+            self.log.debug(f"Metadata json written to '{metadata_path}'")
```

Comment:
Technically the file at that point in time is not completely written yet so the phrasing in the log message is a bit odd.

```suggestion
            self.log.debug(f"Writing metadata json to '{metadata_path}'")
```

Reply:
> I also think it may be worth it to put it before the open() call so that if whatever error occurs on accessing the filepath the pathis logged.

Yes, please.